### PR TITLE
boinccmd: show CPU/GPU resources in --get_task

### PR DESCRIPTION
**Pull request by davidpanderson**
Also only show relevant info.
E.g. don't show exit status of a job that hasn't finished.

### CODE REVIEW

File: lib/gui_rpc_client_print.cpp [hunk 1]
```diff
@@ -160,26 +160,37 @@ void RESULT::print() {
     foo = (time_t)report_deadline;
     printf("   report deadline: %s", ctime(&foo));
     printf("   ready to report: %s\n", ready_to_report?"yes":"no");
-    printf("   got server ack: %s\n", got_server_ack?"yes":"no");
```
Comment (TheAspens) [inline lib/gui_rpc_client_print.cpp]: Was the line printf("   got server ack: %s\n", got_server_ack?"yes":"no"); intentionally dropped?

**Comment by TheAspens** [comment]
I've tested this and it looks fine.  Once I get an answer to the comment I made above I will be able to merge.

**Comment by davidpanderson** [comment]
It was intentionally dropped.  Once a result is acked by the server,
it's garbage-collected within a few seconds.

**Comment by TheAspens** [comment]
Ok thanks for the clarification.  I'm approving and merging.

**Pull request merged**